app/views/pfmps/show.html.haml

```diff
@@ -16,9 +16,18 @@
           jours
         travaillés
 
+    - if @pfmp.in_state?(:completed) and current_user.can_validate?
+      -if @pfmp.can_be_validated?
+        = form_with scope: :validation, method: :post, url: validate_class_schooling_pfmp_path(@classe, @schooling, @pfmp) do |form|
```

Comment:
ça fait quoi le `scope` ?

Reply:
From the documentation of ActionView: `# Adding a scope prefixes the input field names`

Reply:
oui mais pourquoi on a soudainement besoin ?

Reply:
Tu as raison, visiblement c'est pour éviter l'ambiguité en cas de fields similaires mais sur cette page ca ne sert à rien.